### DISCUSSION TITLE
How do I make nuxt3 into a websocket server?

### DISCUSSION THREAD

**Discussion by johanalkstal** [nuxt/nuxt#18133] [Questions]
I'm new to nuxt, never used version 2 and just now started looking into version 3.

I want to make the nuxt server have a websocket server that another client application can send messages to.
These messages should then be sent to the vue app nuxt is serving, so the vue app needs to be a websocket client.

Basically from somewhere messages should be sent to the nuxt server, that then passes them forward to the frontend app.

How do I go about making this possible? Can I integrate socket.io somehow? Should I?

Any help is appreciated!

**Comment by wobsoriano** [comment]
This is how I implemented it https://github.com/nuxt/framework/discussions/1189

But I believe there's a better way
